docs/association/legal/Privacy_policy.md

```diff
@@ -1,63 +1,90 @@
-# Aragon Privacy Policy
+# Privacy Policy
 
-*Aragon contact details: contact@aragon.one*
+This document sets the privacy rules concerning the use of Aragon Association (the “Aragon”) sites and services, including 
 
-*Mail for requests and queries on privacy: contact@aragon.one*
+https://aragon.org;  
+https://mainnet.aragon.org;  
+https://rinkeby.aragon.org;
+https://forum.aragon.org;  
+https://wiki.aragon.org;  
+https://anj.aragon.org;  
+https://transparency.aragon.org  
 
-## Personal data we process
+Collectively (the “Sites”) that the user agrees with, by using the Sites:
 
-Aragon only processes personal data regarding:
+## 1. What information is collected? 
 
-- Aragon users: professional contact data as necessary to manage and render our services. 
+1.1. Aragon may collect the following user’s personal data: IP address, username, email, crypto wallet addresses, voting history, and DAO addresses you subscribe to. 
```

Comment:
1.1. Aragon may collect the following user’s personal data: IP address, username, email, crypto wallet addresses, voting history, and subscribed DAO addresses.